### ISSUE TITLE
"Quote type" ignored

### ISSUE THREAD

**Issue by the0neWhoKnocks** [bryceosterhaus/atom-jest-snippets#8]
Doesn't seem that https://github.com/bryceosterhaus/atom-jest-snippets/pull/6 works. The default is set to `'`, I've restarted after I installed, and am still seeing `"` being used. Looking at the code in https://github.com/bryceosterhaus/atom-jest-snippets/pull/6/commits/e28a65b5fc81bced7cae387bdd1ae6b8829c1153 seems to imply that the quote type is never used, and it just always uses the hard-coded double quote.

**Comment by bryceosterhaus** [comment]
You're right, looks like we aren't generating the snippets correctly. We need to replace the hard coded quotes with `quoteType`

Would you mind sending a PR for this fix?
